src/test/regress/expected/multi_colocation_utils.out

```diff
@@ -266,11 +266,11 @@ SELECT shards_colocated(1300000, 1300020);
 (1 row)
 
 -- check co-located table list
-SELECT UNNEST(get_colocated_table_array('table1_group1'))::regclass;
+SELECT UNNEST(get_colocated_table_array('table1_group1'))::regclass ORDER BY 1;
```

Comment:
Uuuuggghhh 👎… we now have this change in three separate branches. How did this get checked in in the first place?!?

Lemme merge #1667 and you can rebase on that and remove this commit.

Reply:
Got the shipit before your PR I guess. Waiting for your merge.